util/buildRelease/gen_release

```diff
@@ -166,6 +166,7 @@ SystemOrDie("make man-chpldoc");
 SystemOrDie("make clobber");
 
 print "Creating the examples directory...\n";
+SystemOrDie("rm -rf examples"); # remove examples symbolic link
```

Comment:
I was thinking 'rm examples' -- I'm having trouble seeing what additional benefit the '-f' would give us other than hiding errors/failures that we might want to know about?  But I may not recall what -f does properly (and am not taking the time to look it up on release day).